README.md

```diff
@@ -345,11 +345,18 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
 * [Implement customized TrainingService](docs/en_US/TrainingService/HowToImplementTrainingService.md)
 
 
-## **External Repositories**
+## **External Repositories and References**
 Now we have some external usage examples run in NNI from our contributors. Thanks our lovely contributors. And welcome more and more people to join us!
```

Comment:
With authors' permission, we listed a set of NNI usage examples and relevant articles.

Reply:
fixed